devenv/checks/check_github_access.py

```diff
@@ -0,0 +1,36 @@
+import sys
+from os import path
+from typing import Set, Tuple
+
+sys.path.append(path.abspath(".local/share/sentry-devenv/devenv"))
+from devenv.lib import github  # type: ignore
+
+tags: Set[str] = {"github"}
+name = "Check Github Access"
+
+
+def check() -> Tuple[bool, str]:
+    result = github.check_ssh_access()
+    if result:
+        return True, "You have access to Github"
+    return False, "You do not have access to Github"
```

Comment:
P4: this can definitely wait till v2

To type this a bit more elegantly:

```suggestion
def check() -> doctor.Result:
    result = github.check_ssh_access()
    if result:
        return doctor.Success("You have access to Github")
    return doctor.Fail("You do not have access to Github")
```

`doctor.py`
```
from dataclasses import dataclass


@dataclass
class Result:
    message: str
    success: bool

@dataclass
class Success(Result):
    success: bool = True

    def __str__(self):
        return f"✅ \033[32;1m{self.message}\033m"

@dataclass
class Fail(Result):
    message: str
    success: bool = False

    def __str__(self):
        return f"❌ \033[31;1m{self.message}\033m"
```